src/lib/analytics.js

```diff
@@ -58,7 +65,27 @@ export function trackError(error, message = '', fatal = false) {
  * See: https://github.com/GoogleChrome/web-vitals#using-analyticsjs
  * @param {Object} metric
  */
-function sendToGoogleAnalytics({name, delta, id}) {
+function sendToGoogleAnalytics({name, delta, id, attribution, navigationType}) {
+  let webVitalInfo = '(not set)';
+
+  switch (name) {
+    case 'CLS':
+      webVitalInfo = attribution.largestShiftTarget;
+      break;
+    case 'FID':
+    case 'INP':
+      webVitalInfo = attribution.eventTarget;
+      break;
+    case 'LCP':
+      webVitalInfo = attribution.element;
+      break;
+    case 'TTFB':
+      webVitalInfo = attribution.connectionTime;
```

Comment:
I'm not sure we need to track debug info for TTFB and FCP, but if we are going to—and since this is being set on a custom dimension param—we should pick a dimension (string value) rather than a metric (numeric value).

We *could* create custom metrics for `connectionTime`, `firstByteToFCP`, etc., but unless we have a plan to build reports with that data, I think we should hold off to keep things simple (we can always add it later).

(BTW, when we update to GA4, I definitely think we should include all of this info.)

Reply:
And they're gone!